upgrade/sql/8.1.0.sql

```diff
@@ -39,12 +37,10 @@ SELECT pa.id_product_attribute, l.id_lang, '', ''
 FROM `PREFIX_product_attribute` pa CROSS JOIN `PREFIX_lang` l;
 
 /* Add default redirect configuration */
-INSERT INTO `PREFIX_configuration` (`name`, `value`, `date_add`, `date_upd`) VALUES
-  ('PS_PRODUCT_REDIRECTION_DEFAULT', '404', NOW(), NOW()),
-  ('PS_MAINTENANCE_ALLOW_ADMINS', 1, NOW(), NOW()),
-  ('PS_AVIF_QUALITY', '90', NOW(), NOW()),
-  ('PS_IMAGE_FORMAT', 'jpg', NOW(), NOW())
-;
+/* PHP:add_configuration_if_not_exists('PS_PRODUCT_REDIRECTION_DEFAULT', '404'); */;
+/* PHP:add_configuration_if_not_exists('PS_MAINTENANCE_ALLOW_ADMINS', '1'); */;
```

Comment:
Not sure this is important, but I see the original value was an integer.

Reply:
It will behave the same when inserting it into the database, but I can change it if you want me to.

Reply:
If you're certain the result will be the same, no need to change it.